src/lib/replication/service/raft_repl_service.cpp

```diff
@@ -264,6 +306,11 @@ void RaftReplService::load_repl_dev(sisl::byte_view const& buf, void* meta_cooki
         return;
     }
 
```

Comment:
Do you mean use `repl_dev->is_destroy_pending()` instead of the check `destroy_pending == 0x1`. That is not possible because repl_dev is not created yet, we need to create repl_dev based on rd_sb value.